packages/apollos-data-prayer/src/prayer-requests/resolver.js

```diff
@@ -10,6 +10,25 @@ export default {
       dataSources.PrayerRequest.getFromCurrentPerson(),
     getPrayerRequestsByGroups: (root, args, { dataSources }) =>
       dataSources.PrayerRequest.getFromGroups(),
+    savedPrayers: async (root, args, { dataSources }) => {
+      try {
+        const followings = await dataSources.Followings.getFollowingsForCurrentUser(
+          {
+            type: 'PrayerRequest',
+          }
+        );
+
+        const stuff = await followings.get();
+        const prayerIds = stuff.map((follow) => follow.entityId);
+        const prayers = await dataSources.PrayerRequest.getFromIds(
+          prayerIds
+        ).get();
+
+        return prayers;
+      } catch (err) {
+        throw new Error(err);
```

Comment:
```suggestion
        throw new Error(`${err} - Possibly not logged in!`);
```

so it atleast lets them know

<img width="1374" alt="Screen Shot 2019-05-01 at 11 26 39 PM" src="https://user-images.githubusercontent.com/2659478/57055355-f4bdf880-6c68-11e9-9ae2-9289d0b87868.png">

Reply:
I don't like that.

Reply:
I think we're grasping at straws here. Let's not hold up on error messaging when we have a ton of work to do to launch beta.